src/sgame/sg_bot_nav.cpp

```diff
@@ -845,6 +845,48 @@ static int WallclimbStopTime( gentity_t *self )
 	return self->botMind->lastNavconTime + self->botMind->lastNavconDistance * 4 + 500;
 }
 
+// check if a wall climbing bot is running into an attackable human entity, if so: attack it
+static void BotMaybeAttackWhileClimbing( gentity_t *self )
+{
+	ASSERT( self->botMind->lastNavconDistance > 0 );
+
+	int ownClass = self->client->ps.stats[ STAT_CLASS ];
+	glm::vec3 playerMins, playerMaxs;
+	BG_BoundingBox( static_cast<class_t>( ownClass ), &playerMins, &playerMaxs, nullptr, nullptr, nullptr );
+
+	glm::vec3 origin = VEC2GLM( self->s.origin );
+	glm::vec3 dir = glm::vec3( 0.0f, 0.0f, 1.0f );
```

Comment:
Walls are not necessarily exactly vertical. I think it would make more sense to use the direction the bot is facing

Reply:
Yes